GoogleMapsApi.Test/IntegrationTests/BaseTestIntegration.cs

```diff
@@ -12,39 +14,41 @@ namespace GoogleMapsApi.Test.IntegrationTests
 
     public class BaseTestIntegration
     {
-        private string apiKey;
+        private string _apiKey;
```

Comment:
Why did you change the naming convention? In this project we use C# naming convention means private variables are camleCase. https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/inside-a-program/coding-conventions

Reply:
The Visual Studio 2019 don't really need the file for package to nuget format. It is included in the csproj file.

Reply:
Would not touch naming as well.